flutter/lib/src/feedback/sentry_feedback_widget.dart

```diff
@@ -0,0 +1,266 @@
+// ignore_for_file: library_private_types_in_public_api
+
+import 'package:flutter/material.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+class SentryFeedbackWidget extends StatefulWidget {
+  const SentryFeedbackWidget({
+    super.key,
+    this.associatedEventId,
+    this.hub,
+    this.title = 'Report a Bug',
+    this.nameLabel = 'Name',
+    this.namePlaceholder = 'Your Name',
+    this.emailLabel = 'Email',
+    this.emailPlaceholder = 'your.email@example.org',
+    this.messageLabel = 'Description',
+    this.messagePlaceholder = 'What\'s the bug? What did you expect?',
+    this.submitButtonLabel = 'Send Bug Report',
+    this.cancelButtonLabel = 'Cancel',
+    this.isRequiredLabel = '(required)',
```

Comment:
I would probably encapsulate this in a class. That make it a tad easier to deal when adding new fields/translations.

Reply:
IMHO: It's easier for the user to just provide individual ctor params, it's the same on JavaScript. Also not really easier or harder to maintain either way.